.github/workflows/tests.yml

```diff
@@ -31,9 +31,10 @@ jobs:
 
       # run tests
       - name: run unit tests
-        run: go test -race -timeout 30m -v -p 2 ./...
-  test-cmd-ibctest:
-    name: test-cmd-ibctest
+        # -short flag purposefully omitted because there are some longer unit tests
+        run: go test -race -timeout 10m -p 2 $(go list ./... | grep -v /cmd | grep -v /examples)
```

Comment:
It seems like we are expecting some longer running tests here but the timeout value changes from `30m` to `10m`, does that make sense or should we use the larger value still?